src/utils/SpokeUtils.ts

```diff
@@ -70,7 +72,7 @@ export function populateV3Relay(
  *        // where the deposit with deposit ID = targetDepositId was created.
  */
 export async function getBlockRangeForDepositId(
```

Comment:
In general we should probably mark this as a function that only works for deposits with a sequential depositId.

We might also want a utility that accepts a deposit instance and returns true/false depending on whether the depositId is > 2³²-1. Then we could use it in this function to apply special handling if it ever gets requested to resolve a non-sequential depositId.

It'd be nice to assert() to catch bugs, but since this function would also deal with dynamic input, that would pose a potential availability problem. So it might instead be nice if we could warn loudly via the logger.

wdyt?

Reply:
I agree with everything you said above but I'm not sure why we can't use `assert(depositId <  2³²-1)`?

Reply:
For now, I've opted to just throw errors if we have an unsafe deposit ID (since we don't have a logger in utils) so that we can handle it upstream.

Reply:
I'm still a bit nervous that this might cause an outage in production under some unforseen circumstance. But in the short term it will also help us to clearly identify that, so OK :+1: